lte/gateway/dev_tools.py

```diff
@@ -119,13 +155,34 @@ def _register_agw(network_type: str):
         ),
         connected_enodeb_serials=[],
     )
-    dev_utils.cloud_post(f'{network_type}/{network_id}/gateways', gw_payload)
+    dev_utils.cloud_post(f'lte/{network_id}/gateways', gw_payload)
     print()
     print(f'=========================================')
     print(f'Gateway {gw_id} successfully provisioned!')
     print(f'=========================================')
 
 
+def _deregister_agw(network_type: str):
+    network_id = NIDS_BY_TYPE[network_type]
+    hw_id = dev_utils.get_hardware_id_from_vagrant(vm_name='magma')
+    already_registered, registered_as = dev_utils.is_hw_id_registered(
+        network_id, hw_id,
+    )
+    if not already_registered:
+        print()
+        print(f'===========================================')
+        print(f'VM is not registered (hwid: {hw_id} )')
+        print(f'===========================================')
+        return
+
+    dev_utils.cloud_delete(f'lte/{network_id}/gateways/{registered_as}')
+    print()
+    print(f'=========================================')
+    print(f'AGW Gateway {registered_as} successfully removed!')
+    print(f'(restart AGW services on magma vm)')
+    print(f'=========================================')
+
```

Comment:
i'm tempted to silence these f' related lints.... They can be controlled here: https://github.com/magma/magma/blob/master/setup.cfg

Reply:
let me remove them. I think f-string will not make sense here if htere is no format to be applied at all

Reply:
all done now